core-bundle/src/Command/UserPasswordCommand.php

```diff
@@ -144,18 +149,15 @@ private function askForPassword($label, InputInterface $input, OutputInterface $
     /**
      * Validates the password length and creates the password hash.
      *
+     * @param Config $config
      * @param string $password
      *
      * @throws InvalidArgumentException
      *
      * @return string
      */
-    private function validateAndHashPassword($password)
+    private function validateAndHashPassword($config, $password)
```

Comment:
```suggestion
    private function validateAndHashPassword(Config $config, $password)
```

But I actually would prefer `int $minLength` here. WDYT?

Reply:
I wasn't sure if I could type hint the adapter? :-)

Do you mean `?int $minLength` or would you set the default outside the method as well? (I think it should stay in there but the method already has multiple responsibilities).

Reply:
I mean

```php
private function validateAndHashPassword($password, $minLength = 0)
{
}
```

or maybe even `$minLength = 8`.

Reply:
Have a look :-)